ui/packages/catalog-ui-search/src/main/webapp/component/location-new/utils/dms-utils.js

```diff
@@ -422,16 +422,12 @@ const lon = {
 }
 
 const validateDmsLatInput = input => {
-  if (
-    input.slice(lat.degreesBegin, lat.degreesEnd) > 90 ||
-    input.slice(lat.minutesBegin, lat.minutesEnd) > 60 ||
-    input.slice(lat.secondsBegin, lat.secondsEnd) > 60
-  ) {
-    return false
-  } else if (input.slice(lat.minutesBegin, lat.minutesEnd) === '60') {
-    return input.slice(lat.degreesBegin, lat.degreesEnd) + '°60\'00"'
-  } else if (input.slice(lat.degreesBegin, lat.degreesEnd) === '90') {
+  if (input.slice(lat.degreesBegin, lat.degreesEnd) > 90) {
     return '90°00\'00"'
```

Comment:
I see a few literals in this file like  `'90°00\'00"'` that are defined as constants in location.js. if it makes sense you could reference them here

Reply:
Did not know those were constants. Thanks!!